src/components/Button.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledButton = styled.button`
+  background: ${p => p.isGradient ? 'linear-gradient(92.58deg, #0DEFE1 0%, #78FF96 100%)' : p.backgroundColor};
+  color: ${p => p.textColor};  
+  height: ${p => p.height};
+  width: ${p => p.width};
+  border-radius: ${p => p.borderRadius};
+  border: none;
+
+  ${(p) =>
+    p.isHover &&
+    `&:hover {
+      cursor: pointer;
+      color: #224B5C;
+      background: none;
+    }`
+  }
+`;
+
+export default function Button({
+  backgroundColor,
+  textColor,
+  height,
+  width,
+  borderRadius,
+  children,
+  fontSize,
+  isGradient = false,
+  isHover = false }) {
```

Comment:
Do we need this functionality? I feel like the buttons on the site will always have a hover state right?

Reply:
Oh true... would this be a problem for the live portal button though? Since that should have at least the cursor be a pointer on hover right - can we add that as a default styling and leave the rest as is?